spec/requests/entrypoint_spec.rb

```diff
@@ -40,18 +40,21 @@
   it "returns server_info" do
     api_basic_authorize
 
-    get api_entrypoint_url
-
-    expect(response.parsed_body).to include(
-      "server_info" => a_hash_including(
-        "version"     => Vmdb::Appliance.VERSION,
-        "build"       => Vmdb::Appliance.BUILD,
-        "appliance"   => MiqServer.my_server.name,
-        "server_href" => api_server_url(nil, MiqServer.my_server),
-        "zone_href"   => api_zone_url(nil, MiqServer.my_server.zone),
-        "region_href" => api_region_url(nil, MiqRegion.my_region)
+    Timecop.freeze(Time.new(2018, 0o1, 0o1, 0o0, 0o0, 0o0).utc) do
+      get api_entrypoint_url
+
+      expect(response.parsed_body).to include(
+        "server_info" => a_hash_including(
+          "version"     => Vmdb::Appliance.VERSION,
+          "build"       => Vmdb::Appliance.BUILD,
+          "appliance"   => MiqServer.my_server.name,
+          "time"        => Time.now.utc.iso8601.to_s,
```

Comment:
With time frozen, you can now make an assertion on a literal, i.e. `"2018-01-01T00:00:00.00Z"`